auth_service/app.py

```
@@ -0,0 +1,91 @@
+from flask import Flask, request, jsonify
+from flask_cors import CORS  # Import CORS
+from utils import hash_password, generate_token
+import sqlite3
+import json
```

Ruff (0.8.0)

`json` imported but unused

Remove unused import: `json`

(F401)